### DISCUSSION TITLE
READJUSTMENT UNBALANCED SOLAR FLARE SEASONAL EVENT (Mine 10 Stone, Harvest 80 Beetroots)

### DISCUSSION THREAD

**Discussion by JustJayHSP** [sunflower-land/sunflower-land#2319] [Ideas]
Hello Good Day Everyone,

To anyone wondering why is the Solar Flare Seasonal Event unbalanced is because of the 2 Specific Quest we received on the first day.

**10 Mine Stone
Harvest 80 Beetroots**

Now you might be wondering what about them?

It's because if you have a Fully Expanded Land yet you've gotten the least amount of (38-39)Crop Plots and (8-9 Stone Nodes)Nodes you are **most likely already behind the race.**

People with **38-39 Farm Plots CANNOT plant 2 Cycles of Beetroots to finish Harvest 80 Beetroots**. No these players must do a 3rd cycle or pre-plant an extra 2-4 Beetroot before the Quest Came in. Not a Bad Workaround right?

But what about 10 Stone Nodes? People with **8-9 Stone Nodes with Fully Expanded Farms CANNOT mine 10 Stones in 1 whole cycle** this makes you wait for an extra 3 and a half or 4 hours longer than anyone who has 40-48 Farm Plots and 10-12 Stone Nodes.

This is now an unfair event for people who are serious enough to mint Cabbage Girl, **players who have the least amount of nodes and farm crops are FIGHTING A LOSING BATTLE.**

If in the future, Developers would take note of the Fully Expanded Farm with Least amount of Farm Plots and Nodes and readjust the future Quest for Solar Flare or any Events then people will be happy.

The readjustment would as follows:
Mine 8 Stones
Harvest 75 Beetroots

This way even if you're the unluckiest fellow of all time you still have a chance at getting enough Solar Flare Tickets to mint the NFTs you want.
![335045884_107725542238799_6495727978537188533_n](https://user-images.githubusercontent.com/107456378/223703427-2bde5b9a-242f-4103-acda-48ca61fbf2ac.jpg)

**Comment by LandLadyFarmer** [comment]
sure i found another thing thats unfair, lots of people have more time to farm than me so that must also be fixed so it's not unfair..
Then we have that i'm old and don't have the stamina to farm like others so we need to pause the game for 8 hours every night (my time zone) or else it's so unfair....
Come on!!!! how far should take everything that's happens in the game in form of fairness,, this was little to much for me to take it 
and i really hope devs don't listen to this.....
Should we set the limit down by those that have 2 expansions???
Makes me sad to read things like this, real childish behavior.....

Most people be done or started so fix it now it's wrong

**Comment by VZozul** [comment]
Interesting and sad. It's too much, don't you think? The problem is that those who have a lot of resources all this time had more than others. In the balance discussions, it seems like they decided that they will equalize the resources, they will take from those who have more, but will compensate them for this. Seriously? Not only have they been mining more all this time, but also compensation? And then this quest, in which they will also win! This is brilliant, keep it up, justice at every turn. For reference, 4+ gold nodes - 942 lands, 5+ - 260 lands, I have a full expansion and only 3 gold nodes, unless of course the rumors about the gold quest are true. In general, this is sad guys. It was necessary to do the opening of the season after balancing.

**Comment by offerT** [comment]
The number of tasks should be more reasonable, and the difficulty gap should not be too large

**Comment by Myth727** [comment]
I firmly believe we should have a skip button atleast once a day. Let's say a task is too hard, time consuming or costly for a user. Then they should be allowed to skip 1 per day. Just my 2 cents.

**Comment by paduv** [comment]
You can add 'Mine 5 gold' to the list. With 4 nodes or less, you're punished by 24 hours, compared to someone with the regular 5 nodes (at max expansion).
AND this task appears 2 times in the list ! So, with my 4 nodes, i'm 48 hours late just because of this.. (i also only have 38 plots, so it adds up...)

I've been waiting for the balancing to make up for that, but it will come too late, so i'll probably miss the cabbage girl by a thread..

Balancing is even more important when we have this race for rare nft's. It's not just about a few less resources, it's about the main feature of the game: getting nft's (as in 'play-to-own')

I believe there should be a one-time Ticket airdrop for people with fewer-than-normal resources at maxed expansions.
Ideally, it should come at the same time as the balancing itself, & ideally it should be tomorrow or before the end of the week.
The number of Tickets to be airdropped should be calculated looking at the time lost, compared to full-noded players. So, if you have less than 5 gold nodes, you lost 48 hours, you should get back 2 days worth of quest-completing Tickets. I don't know what's the average per day but you get the point.

**Reply by rasheed19871** [reply]
5 gold was my problem too because I only have 4 nodes, that the reason I am behind

**Comment by ghost** [comment]
I encourage you to explore my solution to the quest balance problem:  https://github.com/sunflower-land/sunflower-land/discussions/2326
With this quest system, all players will be able to do daily quests and get tickets, which will have a very good effect on gameplay.
